endpoints/citrus-http/src/main/java/org/citrusframework/http/interceptor/LoggingClientInterceptor.java

```diff
@@ -120,6 +121,24 @@ public boolean hasMessageListeners() {
      * @return
      */
     private String getRequestContent(HttpRequest request, String body) {
+        String contentType = request.getHeaders().getFirst(HttpHeaders.CONTENT_TYPE);
+        if (contentType != null) {
+            String[] contentTypeParts = contentType.split(";");
+            for (String contentTypePart : contentTypeParts) {
+                if(contentTypePart.startsWith("charset=")) {
+                    String charset = contentTypePart.split("=")[1];
+                    try {
+                        body = new String(body.getBytes(), charset);
+                    } catch (UnsupportedEncodingException e) {
+                        body = new String(body.getBytes(), StandardCharsets.UTF_8);
```

Comment:
please use static imports wherever possible.

```suggestion
                        body = new String(body.getBytes(), UTF_8);
```

Reply:
added static imports